app/src/test/java/mozilla/lockbox/store/AutoLockStoreTest.kt

```diff
@@ -89,11 +90,8 @@ class AutoLockStoreTest {
 
     private val lockingSupport = spy(TestLockingSupport())
 
-    val subject = AutoLockStore(
-        settingStore = settingStore,
-        lifecycleStore = lifecycleStore,
-        dataStore = dataStore,
-        dispatcher = dispatcher
+    val subject = AutoLockSupport(
+        settingStore = settingStore
```

Comment:
Going from needing settings, lifecycle, datastore, and dispatcher to _just_ the settingStore ❤️